### PR TITLE
Move the external DB installer parameters around

### PR DESCRIPTION
**Pull request by ekohl**
This way they are all consistent with eachother. They follow:

* manage
* host
* name
* user
* password

Missing parameters were added for consistency.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc [hunk 1]
```diff
@@ -11,20 +11,22 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
...
---foreman-db-database foreman \
---foreman-db-host _postgres.example.com_ \
---foreman-db-manage false \
---foreman-db-password _Foreman_Password_ \
---foreman-proxy-content-pulpcore-manage-postgresql false \
---foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
---foreman-proxy-content-pulpcore-postgresql-host _postgres.example.com_ \
---foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_ \
---foreman-proxy-content-pulpcore-postgresql-user pulp \
+# {forman-installer} \
```
Comment (maximiliankolb) [inline guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc:14]:
```suggestion
# {foreman-installer} \
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
typo in attribute name.

File: guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc (hunk 1 above)
Comment (ekohl) [reply guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc]: Bleh, that snuck in when rebasing.

**Review by maximiliankolb — APPROVED** [review]
LGTM.

**Review by ehelms — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   a4b868f993..e1bbc8ed4c  3.11 -> 3.11
   f2609aa91b..2ec954e6fe  3.10 -> 3.10

**Pull request merged**